docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,219 @@
+-   `Layouts` are containers used to compose views into logical structures. Some available types are absolute, grid, relative, and stack layouts; each provides mechanisms, such as orientation, spacing, and padding, to control the layout. The `StackLayout` class is a basic layout where you can simply stack controls on top of, or side-by-side, one another. Layouts can also be bundled with and nested into each other.
+-   `Views` are the controls displayed on the user interface, such as labels, buttons, and text entry boxes.
+-   `Cells` are specialized elements used for items in tables and lists, which help describe how the items must be rendered.
+
+The following shows the portable code portion of the Tizen Xamarin.Forms project **MainPage.xaml** file, generated by the template:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="CrossTemplate1.MainPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label Text="Welcome to Xamarin.Forms!"
+                VerticalOptions="CenterAndExpand" 
+                HorizontalOptions="CenterAndExpand" />
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+```
+
+This application is constructed with the following Xamarin.Forms controls:
+
+- There is the `ContentPage` where you set up the view to display. You assign what you want to display to the `Content` property of the ContentPage.
+- The `StackLayout` is added. This layout positions ("stacks") its child elements in a single vertical (default) or horizontal line.
+- The `Label` is added, whose `Text` property is the message you want to display.
+
+The code does not show the instantiation of the `App` class. Since application launching is platform-specific, the launching part, including the instantiation of the `App` class and the declaration of the `Main()` function (required as the entry point of every C\# program), happens in the \<projectname\>.Tizen project, in the matching file there. The following example shows the content of the `<projectname>.Tizen.cs` file:
+
+```csharp
+using System;
+using Xamarin.Forms;
+
+namespace CrossTemplate1
+{
+    class Program : global::Xamarin.Forms.Platform.Tizen.FormsApplication
+    {
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+
+            LoadApplication(new App());
+        }
+
+        static void Main(string[] args)
+        {
+            var app = new Program();
+            Forms.Init(app);
+            app.Run(args);
+        }
+    }
+}
+```
+
+## Package Your Application
+
+A Tizen .NET application is deployed in the form of an installable package, with the package file extension `.tpk`. A Tizen .NET package has a relatively simple structure: internally it is a ZIP file with content that matches the directory layout of the project.
```

Comment:
Slight modification:
```suggestion
A Tizen .NET application is deployed in the form of an installable package, with the package file extension `.tpk`. A Tizen .NET package has a relatively simple structure, internally it is a ZIP file with content that matches the directory layout of the project.

Reply:
This IoT Page is based on Mobile, TV, Wearable pages.
ex) https://docs.tizen.org/application/dotnet/get-started/mobile/first-app/

I think they should have a similar format.

Reply:
@xbldx As suggested by @mijin85cho to maintain consistency across getting started pages, we keeopthe sentence as is.